Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs

```diff
@@ -54,6 +54,7 @@ public static IWebHostBuilder CreateWebHostBuilder<T>(string[] args)
             .ConfigureAppConfiguration((builderContext, config) =>
             {
                 config.AddEnvironmentVariables(prefix: EnvironmentPrefix);
+                config.AddJsonFile("appsettings.local.json", true, true); // Loads local settings
```

Comment:
We need to add that to gitignore as well if the purpose is to store local secrets

Reply:
Not going to implement secrets in this PR, and the local settings was already gitignored